app/models/manageiq/providers/kubernetes/container_manager/event_catcher_mixin.rb

```diff
@@ -1,5 +1,9 @@
 module ManageIQ::Providers::Kubernetes::ContainerManager::EventCatcherMixin
   extend ActiveSupport::Concern
+  ENABLED_EVENTS = {
+    'Node' => %w(NodeReady NodeNotReady rebooted),
+    'Pod'  => %w(scheduled failedScheduling failedValidation HostPortConflict created failed started killing)
```

Comment:
@dkorn sorry this is used for `ENABLED_EVENTS` so we'll have to keep it as it is.